src/SFA.DAS.PSRService.Web/wwwroot/assets/stylesheets/esfa-govuk-base.css

```diff
@@ -742,6 +746,81 @@ hr {
       background-image: url("../images/icon-pointer-2x.png");
       background-size: 30px 19px; } }
 
+.button-secondary {
```

Comment:
I think these changes should be made in the site.css in wwwroot/css

Reply:
Done